sdk/helper/aliasmetadata/alias_metadata_acc_test.go

```diff
@@ -0,0 +1,497 @@
+package aliasmetadata
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func TestAcceptance(t *testing.T) {
```

Comment:
Can this be made into a table test? It's doing a lot of different calls to `HandleRequest` in a single test function.

Reply:
Broke this into multiple tests in https://github.com/hashicorp/vault/pull/8783/commits/6d19459b96ae327fb1040049a9591b81562d4551